pkg/tanka/load.go

```diff
@@ -114,6 +124,9 @@ type Loader interface {
 	// List returns metadata of all possible environments at path that can be
 	// loaded
 	List(path string, opts LoaderOpts) ([]*v1alpha1.Environment, error)
+
+	// EvalJsonnet returns the raw evaluated Jsonnet
+	EvalJsonnet(path string, opts LoaderOpts) (interface{}, error)
```

Comment:
```suggestion
	Eval(path string, opts LoaderOpts) (interface{}, error)
```

All other methods of the interface are single verbs and Loaders operate on Jsonnet anyways, so this seems redundant.

Reply:
This will conflict with the Eval functions in `pkg/tanka/workflow.go`.